CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Fix: [#396] Preferred owner name is not saved.
 - Fix: [#423] Date in challenge tooltip is incorrect.
 - Fix: [#425] Changing resolution in fullscreen mode doesn't work.
+- Fix: [#428] Show invalid build information in build vehicle window. Original bug.
```

Comment:
This makes it sound like it's showing invalid information, but in effect it's showing an extra error message, right? Could you clarify?

Reply:
Its an extra error message when you can't build due to the vehicle being invalid.

Reply:
Does this cover it, then?
```suggestion
- Fix: [#428] Show an error when a vehicle can't be built due to invalid properties. (Original bug.)
```